sdks/python/tox.ini

```diff
@@ -320,31 +320,30 @@ commands =
   # Allow exit code 5 (no tests run) so that we can run this command safely on arbitrary subdirectories.
   /bin/sh -c 'pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_pytorch {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
 
-[testenv:py{37,38,39,310}-onnx-113]
+# TODO(https://github.com/apache/beam/issues/25796) - uncomment onnx tox task once onnx supports protobuf 4.x.x
+# [testenv:py{37,38,39,310}-onnx-113]
 # TODO(https://github.com/apache/beam/issues/25443)
 # apparently tox has problem when substitution key has single value. Change back to -onnx-{113,...}
 # when multiple onnx versions are tested.
-deps =
-  onnxruntime==1.13.1
-  pandas==1.5.2
-  torch==1.13.1
-  tensorflow==2.11.0
-  tf2onnx==1.13.0
-  skl2onnx==1.13
-  transformers==4.25.1
-extras = test,gcp
-commands =
-  # Log onnx version for debugging
-  /bin/sh -c "pip freeze | grep -E onnx"
-  # Run all ONNX unit tests
-  pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_onnx {posargs}
+# deps =
+#  onnxruntime==1.13.1
+#  pandas==1.5.2
+# torch==1.13.1
+# tensorflow==2.11.0
```

Comment:
should be TF 2.12.0 too, my guess.

Reply:
In general, it is not common to have large chunks of commented out code in the live codebase. It quickly rots and people tend to delete it when seeing it. One option is to disable the task here: https://github.com/apache/beam/blob/13a7cbf73f9db22fab5b7a8b5377bd8ec73538de/sdks/python/test-suites/tox/py38/build.gradle#L106, which should probably be a 1-line change.

Reply:
looks like this change is already in your PR

Reply:
Yes, I commented the 3 lines here and uncommented the onnx task in `tox.ini`.  Without the commands in tox/py38/build.gradle, the commands of onnx in `tox.ini` would be no-op